packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -13,6 +13,8 @@ import { ContainerUtils } from './container/container-utils';
 import { providerInfos } from '../stores/providers';
 import NoContainerEngineEmptyScreen from './image/NoContainerEngineEmptyScreen.svelte';
 
+setInterval(() => fetchContainers(), 5000);
```

Comment:
it might raised issues if we're doing polling vs event based
if we could do the refresh of the time only on client side, it would be better, WDYT ?
I can also help/contribute

Reply:
Thank you for reviewing. I think so too.
However, I couldn't come up with a good coding method to refresh of the time...

Reply:
@sstosh I've amended your PR with one commit


https://user-images.githubusercontent.com/436777/183963384-f0d8b1b6-7bd8-4dd2-90b6-ef32f0f5885c.mp4

let me know what you think ?

Reply:
Check on client side the uptime (do not ask again on server to fetch all containers)

And we don't poll often if we don't need to.
We refresh uptime every
- 2 seconds if container started since less than one minute
- 1 minute if container started since less than one hour
- 1 hour if container started since less than one day
- else every day

and only if containers are running

Round the value to the single unit (can be second, minute, hours, days)

Reply:
> let me know what you think ?

I think it's a reasonable and good coding method.
Thank you for your help!